tox.ini

```diff
@@ -1,7 +1,9 @@
 [tox]
-envlist = py27, py35
+# to use this install "poetry", "tox" and "tox-poetry"
+# then run tox
+isolated_build = True
+envlist = py37, py38, py39, py310
```

Comment:
@jensens I realized, I removed this in baf859b47331d6bc9edcc10ec91c5de463244b5e 

I will bring it back soon based on my work on another project here: https://github.com/develtech/django-slugify-processor/blob/6e4d8ab/tox.ini

Back at the time of the commit, were these working for you?

Reply:
@jensens I propose trying both the most recent change 8407310 and the one at f477cae6a6c9e1aadd86cf8226f49ce75380967a (from this PR)

- `poetry install -E test`
- `tox`

To test the old one:
- `rm -rf .tox`which 
- `git checkout f477cae6a6c9e1aadd86cf8226f49ce75380967a -- tox.ini`
- `tox`

To try going really far back:

- `git reset --hard <ref>`
- `poetry install -E test`
- `tox`

If it's the older ref, it may be needed to do `poetry env use 3.9` and update `.tool-versions` from `poetry 1.1.7` to `poetry 1.1.12`

Sorry one again for blowing that change away!  I wasn't recalling at the time. It must have been because I was mass updating multiple projects with the same changes